src/BlazorUI/Bit.BlazorUI/Components/Navs/NavBar/BitNavBar.scss

```
@@ -94,3 +209,25 @@
+// to the unfilled look: the Color role on a transparent background.
+@each $role, $tokens in $bit-color-roles {
+    .bit-nbr-a#{$role} {
+        --bit-nbr-clr-acc-hover: #{role($tokens, hover)};
+        --bit-nbr-clr-acc-active: #{role($tokens, active)};
+
+        @if role($tokens, kind) == semantic {
+            // A semantic accent covers the item with a vivid fill, which the Color role is not meant to
+            // sit on, so the whole content of the item moves to the accent's on-color.
+            --bit-nbr-clr-acc-text: #{role($tokens, on)};
+        }
+        @else {
```

Stylelint (17.14.0)

[error] Unexpected newline after "}" of `@if` statement (scss/at-if-closing-brace-newline-after)

(scss/at-if-closing-brace-newline-after)